LEGO1/legoentity.h

```diff
@@ -34,34 +35,34 @@ class LegoEntity : public MxEntity {
 		return !strcmp(name, LegoEntity::ClassName()) || MxEntity::IsA(name);
 	}
 
-	virtual MxResult InitFromMxDSObject(MxDSObject& p_dsObject); // vtable+0x18
-	virtual void Destroy(MxBool p_fromDestructor);               // vtable+0x1c
-	virtual void ParseAction(char*);                             // vtable+0x20
-	virtual void VTable0x24();                                   // vtable+0x24
-	virtual void VTable0x28();                                   // vtable+0x28
-	virtual void VTable0x2c();                                   // vtable+0x2c
+	virtual MxResult InitFromMxDSObject(MxDSObject& p_dsObject);                   // vtable+0x18
+	virtual void Destroy(MxBool p_fromDestructor);                                 // vtable+0x1c
+	virtual void ParseAction(char*);                                               // vtable+0x20
+	virtual void SetROI(LegoROI* p_roi, MxBool p_bool1, MxBool p_bool2);           // vtable+0x24
+	virtual void SetWorldTransform(MxVector3& loc, MxVector3& dir, MxVector3& up); // vtable+0x28
+	virtual void ResetWorldTransform(MxBool p_inVehicle);                          // vtable+0x2c
 	// OFFSET: LEGO1 0x10001090
-	virtual void VTable0x30(undefined4 p_unk50) { m_unk50 = p_unk50; } // vtable+0x30
-	virtual void VTable0x34();                                         // vtable+0x34
-	virtual void VTable0x38();                                         // vtable+0x38
-	virtual void VTable0x3c();                                         // vtable+0x3c
-	virtual void VTable0x40();                                         // vtable+0x40
-	virtual void VTable0x44();                                         // vtable+0x44
-	virtual void VTable0x48();                                         // vtable+0x48
-	virtual void VTable0x4c();                                         // vtable+0x4c
+	virtual void SetWorldSpeed(MxFloat p_speed) { m_worldSpeed = p_speed; } // vtable+0x30
```

Comment:
Preferably we give the parameter the same name as the member

Reply:
> Preferably we give the parameter the same name as the member
Does this apply even for code that was in the 1996 source like this was?

Reply:
Good question. Personally I'd say yes, it's fine if we introduce some minor extra conventions like this, even if that doesn't equal 100% what we see in the 1996 source. Many things were applied inconsistently in the original source, I don't think taking over inconcistency on this level is worthwhile.